run/grpc-hello-world-gradle/build.gradle.kts

```diff
@@ -25,10 +25,8 @@ repositories {
 }
 
 dependencies {
-    implementation(kotlin("stdlib"))
+    implementation(kotlin("stdlib-jdk8"))
```

Comment:
For my own benefit, what is the difference here? Without this, is the most recent JDK version pulled down?

Reply:
Just adds some JDK8 stuff: https://medium.com/@mbonnin/the-different-kotlin-stdlibs-explained-83d7c6bf293